docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -48,11 +48,11 @@ To install and run Semgrep Pro Engine in the CLI, follow these steps:
     ```bash
     semgrep --pro --config "p/default" 
     ```
-1. Optional: We appreciate your help gathering data as we improve Semgrep Pro Engine! If you are fine with sending r2c usage metrics, run the following command:
+1. Optional: If you prefer not to send usage metrics to Semgrep, run the following command:
```

Comment:
Looking for Milan's opinion on this one.

Reply:
Good by me!